test/common/cdp-driver.js

```diff
@@ -70,7 +71,10 @@ function setupLogging(client) {
     });
 
     client.Log.enable();
-    client.Log.entryAdded(entry => process.stderr.write("CDP: " + JSON.stringify(entry["entry"]) + "\n"));
+    client.Log.entryAdded(entry => {
+        logEntries.push(entry["entry"]);
```

Comment:
Is this related to the unit tests? I thought they use `console.log()`. The CDP `Log` is for messages that Chromium itself generates, not from the JS client script.

Reply:
Nevermind, supposedly this is for picking up Resource Errors.